packages/gateway/src/routes/verifyCode/handler.ts

```diff
@@ -0,0 +1,251 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import * as Hapi from '@hapi/hapi'
+import * as Joi from 'joi'
+import {
+  CERT_PRIVATE_KEY_PATH,
+  CERT_PUBLIC_KEY_PATH,
+  CONFIG_SMS_CODE_EXPIRY_SECONDS,
+  CONFIG_SYSTEM_TOKEN_EXPIRY_SECONDS,
+  CONFIG_TOKEN_EXPIRY_SECONDS,
+  JWT_ISSUER,
+  NOTIFICATION_URL,
+  PRODUCTION,
+  QA_ENV
+} from '@gateway/constants'
+import { set, get, del } from '@gateway/features/user/database'
+import * as crypto from 'crypto'
+import { resolve } from 'url'
+import { readFileSync } from 'fs'
+import { promisify } from 'util'
+import * as jwt from 'jsonwebtoken'
+import fetch from 'node-fetch'
+import { unauthorized } from '@hapi/boom'
+import { logger } from '@gateway/logger'
+const publicCert = readFileSync(CERT_PUBLIC_KEY_PATH)
+import * as t from 'io-ts'
+
+interface ICodeDetails {
+  code: string
+  createdAt: number
+}
+
+type SixDigitVerificationCode = string
+
+const cert = readFileSync(CERT_PRIVATE_KEY_PATH)
+
+const sign = promisify(jwt.sign) as unknown as (
+  payload: string | Buffer | object,
+  secretOrPrivateKey: jwt.Secret,
+  options?: jwt.SignOptions
+) => Promise<string>
+
+interface ISendVerifyCodeResponse {
+  userId: string
+  nonce: string
+  mobile: string
+  status: string
+}
+
+interface ISendVerifyCodePayload {
+  phoneNumber: string
+}
+
+export async function storeVerificationCode(nonce: string, code: string) {
+  const codeDetails = {
+    code,
+    createdAt: Date.now()
+  }
+
+  await set(`verification_${nonce}`, JSON.stringify(codeDetails))
+}
+
+export async function generateVerificationCode(
+  nonce: string,
+  mobile: string
+): Promise<SixDigitVerificationCode> {
+  // tslint:disable-next-line
```

Comment:
![image](https://user-images.githubusercontent.com/1206987/151516202-f8fdc168-9a54-4874-b88e-73808768cd8c.png)

Would it make sense to use something like the recommended here?

Reply:
I think Math.random() is the most used and effective method for random value generation. And also is method is used on this project for login verification code.